### ISSUE TITLE
filterBy bug with nested properties

### ISSUE THREAD

**Issue by Stanteq** [danrevah/ngx-pipes#154]
Hi. I have a problem filtering this case:

```
this.data = [ 
     {a: 1, b: {c:2, d:3}},
     {a: 4, b: null},
]

<p>{{ data | filterBy: ['b.c']: 2}}</p> 
```

`Error: Cannot read property 'c' of null`

As a temporary workaround I need to loop it manually first and delete all `null` properties
Thanks

**Comment by danrevah** [comment]
It is a bug indeed.. anyone want to try and fix it ? :)

**Comment by timrasche** [comment]
Process?

**Comment by Nols1000** [comment]
I've created a PR that should fix this bug.

**Comment by timrasche** [comment]
@Nols1000 Thanks a lot. Hopefully the merge takes not 3 months again...

**Comment by danrevah** [comment]
Released in 2.6.0
